pkgs/applications/version-management/git-credential-manager/default.nix

```diff
@@ -36,9 +32,8 @@ buildDotnetModule rec {
   dotnetInstallFlags = [ "--framework" "net8.0" ];
   executables = [ "git-credential-manager" ];
 
-  runtimeDeps = [ fontconfig ]
-    ++ lib.optionals withGuiSupport [ libX11 libICE libSM ]
```

Comment:
This was the only usage of `withGuiSupport`. I think it should be dropped from the argument list.

Reply:
Unless there's still a way (and a need) to implement it.

Reply:
Well I suppose we've bloated the closure a bit when gui support is disabled.  On the flip side we're not bringing in those dependencies on darwin.  I think removing the option is probably best.